addons/build-debian-tar

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
 #!/system/bin/sh
```

Comment:
the second #! is not necessary?

Reply:
the second #! is not necessary?

Reply:
Hmm I believe I tested with that but it failed. Let me try again. I remember there was no /bin/sh in my android but now I see it (I did change the image since then). I'll test without it and if it works I'll remove it.